js/index.js

```diff
@@ -21,8 +21,43 @@
     skillsList.appendChild(skillItem);
  }
 
+ // This is tthe message section
+var messageForm = document.forms.leave_message;
 
+messageForm.addEventListener('submit', function(event) {
```

Comment:
One tip here is to think about organizing your code into smaller functions or sections. For ex all the variables inside the submit event listener could be outside of it, towards the top so that the eventlistener is tidier! Otherwise looking good.